projects/ngx-pagination-data-source/package.json

```
@@ -23,9 +23,9 @@
     "url": "https://github.com/nilsmehlhorn/ngx-pagination-data-source/issues"
   },
   "peerDependencies": {
-    "@angular/common": "^12.0.0",
-    "@angular/core": "^12.0.0",
-    "@angular/cdk": "^12.0.0"
+    "@angular/common": "^13.1.1",
+    "@angular/core": "^13.1.1",
+    "@angular/cdk": "^13.1.1"
```

I'm not entirely sure on this, but I've always set this version to the lowest corresponding Angular major, so that anyone with Angular 13 can use the library without warnings. We build against the most recent version, but the library should be fine for consumption with any Angular 13 (maybe even lower versions, but that'd complicate things).

```suggestion
    "@angular/common": "^13.0.0",
    "@angular/core": "^13.0.0",
    "@angular/cdk": "^13.0.0"
```